scripts/build/TestImpactAnalysis/tiaf.py

```diff
@@ -20,18 +20,20 @@ def is_child_path(parent_path, child_path):
     return os.path.commonpath([os.path.abspath(parent_path)]) == os.path.commonpath([os.path.abspath(parent_path), os.path.abspath(child_path)])
 
 class TestImpact:
-    def __init__(self, config_file, dst_commit, dst_branch, pipeline, branches_of_truth, pipelines_of_truth):
+    def __init__(self, config_file, dst_commit, src_branch, dst_branch, pipeline, branches_of_truth, pipelines_of_truth):
         # Commit
         self.__dst_commit = dst_commit
         print(f"Commit: '{self.__dst_commit}'.")
         self.__src_commit = None
         self.__has_src_commit = False
         # Branch
+        self.__src_branch = src_branch
+        print(f"Source branch: '{self.__src_branch}'.")
         self.__dst_branch = dst_branch
         print(f"Destination branch: '{self.__dst_branch}'.")
         self.__branches_of_truth = branches_of_truth
         print(f"Branches of truth: '{self.__branches_of_truth}'.")
-        if self.__dst_branch in self.__branches_of_truth:
+        if self.__src_branch in self.__branches_of_truth:
             self.__is_branch_of_truth = True
         else:
             self.__is_branch_of_truth = False
```

Comment:
What odes "branches of truth" mean?

Reply:
These are the branches that are the sources of truth for seed data that will be read from other branches/pipelines (seed data will be written if the current branch and pipeline are the sources of truth).

Reply:
Offline discussion: we settled on `is_seeding_branch`/`is_seeding_pipeline` and `seeding_pipelines` and `seeding_branches` as the convention.